docs/conf.py

```diff
@@ -73,6 +72,12 @@
 autodoc_preserve_defaults = True
 autodoc_member_order = "groupwise"
 
+suppress_warning = []
+nitpick_ignore = []
+
+# Exclude drafts from external documentation
+if tags.has("external"):
+    exclude_patterns.extend(["**-draft*","**-internal*"])
```

Comment:
perhaps change this to `"**-draft.*"` etc, so it is clear that we are just excluding the suffix `-draft.ext`

Reply:
This is to also support whole subdirectories. For example, references/http-draft/index.rst
See PR for example: https://github.com/KatanaGraph/katana-enterprise/pull/2102